src/components/GlobalPreferences/Notifications/NotificationsPreferences.js

```diff
@@ -0,0 +1,170 @@
+        secondary={
+          <React.Fragment>
+            <Box heading="Email account">
+              <Tag
+                uppercase={false}
+                background={theme.infoSurface}
+                size="normal"
+                color={theme.content}
+                css={`
+                  ${textStyle('body2')};
+                  border-radius: 3px;
+                `}
+              >
+                {email}
+              </Tag>
+              <Button
+                css={`
+                  margin-top: ${2 * GU}px;
+                `}
+                wide
+                mode="strong"
+                onClick={onLockSettings}
+              >
+                Lock settings
+              </Button>
+              <Button
+                css={`
+                  margin-top: ${2 * GU}px;
+                  justify-content: left;
+                `}
+                wide
+                onClick={onUpdateEmail}
+              >
+                <IconMail />
+                Update email
+              </Button>
+              <Button
+                css={`
+                  margin-top: ${2 * GU}px;
+                  justify-content: left;
+                `}
+                wide
+                onClick={onDeleteEmail}
+              >
+                <IconTrash
+                  css={`
+                    color: ${theme.negative};
+                  `}
+                />
+                Delete your email
+              </Button>
+              <Info
+                css={`
+                  margin-top: ${2 * GU}px;
+                `}
+              >
+                Learn more about our{' '}
+                <Link href="https://aragon.one/email-collection.md">
+                  Email collection policy
+                </Link>
+                .
+              </Info>
+            </Box>
+          </React.Fragment>
+        }
+      />
+    </React.Fragment>
+  )
+}
+
+function EmailNotificationBox({
+  notificationsDisabled,
+  onSwitchNotificationsStatus,
+}) {
+  const handleSwitchNotificationsStatus = useCallback(() => {
+    onSwitchNotificationsStatus(!notificationsDisabled)
+  }, [notificationsDisabled, onSwitchNotificationsStatus])
+
+  return (
+    <Box heading="Signed In With Email">
+      <div
+        css={`
+          display: flex;
+          flex-direction: column;
+          text-align: center;
+        `}
+      >
+        <div>
+          <img
+            src={emailNotifcationIllustration}
+            width={180}
+            height={180}
+            alt=""
+          />
+        </div>
+        <div
+          css={`
+            display: flex;
+            margin-top: ${5 * GU}px;
+            align-items: center;
+          `}
+        >
+          <Switch
+            checked={!notificationsDisabled}
+            onChange={handleSwitchNotificationsStatus}
+          />
+          <span
+            css={`
+              ${textStyle('body1')};
+              margin-left: ${GU}px;
```

Comment:
```suggestion
              margin-left: ${1 * GU}px;
```

Reply:
haha never understand why we are adding a 1, seen this comment several times haha. Why is that?

Reply:
Haha, i think it's better to make explicit the number of grid units we are using even tough it's the same result.

Reply:
If/When we introduce [the babel preprocessor to transform the gu](https://github.com/aragon/aragon-ui/issues/333), we will have to use the `1 * GU` format, as with css, `1px` is not the same as just `px`.